README.md

```diff
@@ -1,100 +1,8 @@
-
-### Running Locally
-
-A few steps to get started:
-
-1. Make sure you have [Docker](https://www.docker.com/products/docker-desktop) installed on your local machine. 
-2. [Signup for a Convox account](https://convox.com/signup) It's free!
-3. Install the [Convox CLI](https://docs.convox.com/development/installation). 
-4. Install the [Convox local Rack](https://docs.convox.com/development/running-locally)
-
-Once you are all setup you can switch to your local rack with ```convox switch local``` and start your local application with ```convox start``` (make sure you are in the root directory).
-
-Now that your app is up and running you will need to run the migrations with a [one-off command](https://docs.convox.com/management/one-off-commands):
-
-```bash
-convox run web python manage.py migrate
-```
-
-And create a super user for the Django Admin
-
-```bash
-convox run web python manage.py createsuperuser
-```
-
-You should now be able to access your application by going to [https://web.django.convox](https://web.django.convox). If you renamed anything you may need to modify your local URL. The format is https://[service name].[app name].convox
-
-### Deploying to production
-
-In order to deploy to production we have to ensure we have completed the following steps:
-
-1. [Signup for a Convox account](https://convox.com/signup)
-2. [Connect an AWS account](https://docs.convox.com/introduction/getting-started#connect-an-aws-account)
-3. [Install an AWS Rack](https://docs.convox.com/introduction/getting-started#install-an-aws-rack)
-4. Make sure your CLI is [logged in](https://docs.convox.com/reference/login-and-authentication) to your Convox account using ```convox login``` and your [CLI Key](https://console.convox.com/account)
-
-Once you are all set here you can see the name of your production rack
-
-```bash 
-convox racks
-```
-
-And switch your CLI to your production rack
-
-```bash
-convox switch [rack name]
-```
-
-Now you can create an empty application in your production rack
-
-```bash
-convox apps create --wait
-```
-
-And you can deploy your application to production (the first time you do this it may take up to 15 minutes to create the necessary resources)
-
-```bash
-convox deploy --wait
-```
-
-Then you can run your migrations and create your super user same as you did against your local rack
-
-```bash
-convox run web python manage.py migrate
-```
-
-```bash
-convox run web python manage.py createsuperuser
-```
-
-Finally you can retrieve the URL from your production application with
-
-```bash
-convox services
-```
-
-
+The following files were added/modified against a brand new sample Django app copied from the [Django 2.1 tutorial](https://docs.djangoproject.com/en/2.1/intro/tutorial01/)
```

Comment:
Can probably drop the "brand new" here, the usage on rails was to imply it was freshly generated